README.md

```diff
@@ -1,19 +1,83 @@
-# Elastic Enterprise Search Connectors
+# Elastic Enterprise Search connectors
 
 ![logo](logo-enterprise-search.png)
 
-The home of Elastic Enterprise Connector Clients. Use connector clients to
-customize connectors such as Workplace Search Content Sources for advanced use
-cases.
+The home of Elastic Enterprise Connector Clients. This repository contains the framework for customizing Elastic Enterprise Search native connectors, or writing your own connectors for advanced use cases.
 
-Note #1: The connector framework is a tech preview feature. Tech preview features
-are subject to change and are not covered by the support SLA of general release
-(GA) features. Elastic plans to promote this feature to GA in a future release.
+[//]: # (The introduction should:)
+[//]: # (- Summarize the tools provided in this repository.)
+[//]: # (- Summarize the procedures and reference information from this readme and any other docs included in this repo.)
+[//]: # (- Introduce any terminology that is our own.)
+[//]: # (- Identify this feature as a technical preview.)
+[//]: # (Then present a TOC to guide the user through the following sections.)
 
-Note #2: The `main` branch of this repository is currently being heavily reworked. If you want to use the
-connector clients now, it would be advisable to check out/fork the [latest stable branch](https://github.com/elastic/connectors-ruby/tree/8.3).
+Note #1: The connector framework is a tech preview feature. Tech preview features are subject to change and are not covered by the support SLA of general release (GA) features. Elastic plans to promote this feature to GA in a future release.
 
-### System Requirements
+## Terminology
+
+* `connector` - generic term used to refer to both native connectors and connector clients.
+* `connector client` - specific light-weight connector implementation, open-code. Not all clients will be supported as native, but we target to have native connectors also offered as clients. Connector clients can be built by Elastic or community built.
+
+## Disambiguation: connectors-ruby and connectors-python
+
+This repository contains connector clients written in Ruby. However, some connectors will also be implemented in Python. The Python connectors are located in a separate repository, [elastic/connectors-python](https://github.com/elastic/connectors-python).
+
+Before getting started, review important information about this feature:
```

Comment:
If you keep the _terminology_ and _disambiguation_ sections, I'd integrate them into this TOC. Right now it looks like the TOC is part of the disambiguation section.

Reply:
I am removing both of those.